lib/manager/npm/post-update/yarn.spec.ts

```diff
@@ -59,7 +59,14 @@ describe(getName(), () => {
         },
         postUpdateOptions: ['yarnDedupeFewer', 'yarnDedupeHighest'],
       };
-      const res = await yarnHelper.generateLockFile('some-dir', {}, config);
+      const res = await yarnHelper.generateLockFile(
+        'some-dir',
+        {
+          YARN_CACHE_FOLDER: 'yarn-cache-folder',
+          YARN_GLOBAL_FOLDER: 'yarn-global-folder',
```

Comment:
Can you use some tmp folder here, eg `/tmp/renovate/cache/yarn-global`

Reply:
Done :smile: